contracts/FIFOMarket.sol

```diff
@@ -182,10 +190,15 @@ contract FIFOMarket is
       "Market: This contract can only receive BridgedPolygonNORI"
     );
 
+    uint256 certificateAmount = (amount * 100) / (100 + _noriFee); // todo calculate noriFee in loop to allow dynamic fees per removal
+    uint256 remainingAmountToFill = certificateAmount;
     uint256[] memory ids = new uint256[](_queueLength());
     uint256[] memory amounts = new uint256[](_queueLength());
     address[] memory suppliers = new address[](_queueLength());
     uint256 numberOfRemovals = 0;
+    Fee memory noriFee = Fee(_noriFeeWallet, 0, 0, true); // todo throw if supplier address == nori fee address
+    feeAddressToFeeIndex[_orderId][_noriFeeWallet] = noriFee;
+    uint256 highestFeeIndex = noriFee.index;
```

Comment:
The reason I didn't was to make it explicit that the nori fee index was the first index. I'm indifferent to either approach

Reply:
i see, yeah not a big deal either way. reading it at first, it was harder to grok that the fee indices were a fresh concept for every order than maybe it would have been if that was set to 0 but I think it's good to leave it how it is